docs/automation-rules.rst

```diff
@@ -0,0 +1,137 @@
+Automation Rules
+================
+
+Automation rules allow you to execute some actions over new versions.
+
+By default when new versions are created,
+they are not activated.
+If you want to activate it and do other type of actions,
+you need to log in your Read the Docs account first.
+
+Creating an automation rule
+---------------------------
+
+#. Go to your project :guilabel:`Admin` > :guilabel:`Automation Rules`
+#. Click on "Add Rule" button
+#. Fill in the fields
+
+   - Description: A description of what the rule does
+   - Match: What versions the rule should be applied to
+   - Version type: What type of versions the rule should be applied to
+   - Action: What action should be applied to matching versions
+
+#. Click "Save"
+
+How do they work?
+-----------------
+
+When a new tag or branch is pushed to your repository,
+Read the Docs creates a new version.
+
+All rules are run over this version in the order they are listed,
+if the version matches the version type and the pattern in the rule,
+an action is executed over that version.
+
+.. note::
+   
+   All actions of the rules that the version matches will be executed over that version.
+
+Predefined matches
+------------------
+
+There are some predefined matches:
```

Comment:
This sentence isn't much value to the reader because it's using vague language, more specific language would help

```suggestion
Automation rules support several predefined version matches:
```